e2e/artefacts/k8s/nad.yaml

```diff
@@ -1,20 +1,12 @@
 apiVersion: "k8s.cni.cncf.io/v1"
 kind: NetworkAttachmentDefinition
 metadata:
-  name: sriov-net1
+  name: nfnet
   annotations:
     k8s.v1.cni.cncf.io/resourceName: intel.com/ipu
```

Comment:
Is this the resourceName settled by the device plugin? Just to double check

Reply:
> Is this the resourceName used by the device plugin? Just to double check
No, this is just name of the NAD.